plugins/package-managers/node/src/funTest/kotlin/yarn/YarnFunTest.kt

```diff
@@ -54,7 +54,7 @@ class YarnFunTest : StringSpec({
         val definitionFile = getAssetFile("projects/synthetic/yarn/workspaces/package.json")
         val expectedResultFile = getAssetFile("projects/synthetic/yarn/workspaces-expected-output.yml")
 
-        val result = YarnFactory.create().collateMultipleProjects(definitionFile).withResolvedScopes()
+        val result = analyze(definitionFile.parentFile, packageManagers = setOf(YarnFactory())).getAnalyzerResult()
```

Comment:
There are ore places in the code base that call the combination of `create()` / `collateMultipleProjects()` / `withResolvedScopes()` (e.g. `Yarn2FunTest`, `TychoFunTest`). Can we make use of the function there as well?